nntrainer/tensor/float_tensor.cpp

```diff
@@ -293,6 +293,37 @@ TensorV2 &FloatTensor::apply(std::function<float(float)> f,
   return output;
 }
 
+void FloatTensor::copy(const TensorV2 &from) {
+  reshape(from.getDim());
+  copy(from.getData<float>());
+}
+
+void FloatTensor::copyData(const TensorV2 &from) {
+  if (!contiguous) {
+    throw std::runtime_error("Cannot copy non-contiguous tensor");
+  }
+
+  if (size() != from.size())
+    throw std::invalid_argument("Size of tensor to copy must match");
+
+  switch (from.getDataType()) {
+  case ml::train::TensorDim::DataType::FP32:
+    copy(from.getData<float>());
+    break;
+  case ml::train::TensorDim::DataType::FP16:
+/// @todo remove #ifdef ENABLE_FP16
```

Comment:
I think you can remove this in this PR.

Reply:
this case is fp16 --> float. float --> fp16 case will occur in the HalfTensor class.